packages/superset-ui-legacy-preset-chart-deckgl/src/utils/computeBoundsFromPoints.ts

```diff
@@ -0,0 +1,47 @@
+import { extent as d3Extent } from 'd3-array';
+
+const PADDING = 0.25;
+const GEO_BOUNDS = {
+  LAT_MAX: 90,
+  LAT_MIN: -90,
+  LNG_MAX: 180,
+  LNG_MIN: -180,
+};
+
+/**
+ * Get the latitude bounds if latitude is a single coordinate
+ * @param latExt Latitude range
+ */
+function getLatBoundsForSingleCoordinate(latExt: [number, number]) {
+  const latMin =
+    latExt[0] - PADDING < GEO_BOUNDS.LAT_MIN ? GEO_BOUNDS.LAT_MIN : latExt[0] - PADDING;
+  const latMax =
+    latExt[1] + PADDING > GEO_BOUNDS.LAT_MAX ? GEO_BOUNDS.LAT_MAX : latExt[1] + PADDING;
+
+  return [latMin, latMax];
+}
```

Comment:
Can do

```js
import { Point } from './getPointsFromPolygon';  // or './common'

function getLatBoundsForSingleCoordinate([latMin, latMax]: Point) {
  return [
    Math.max(GEO_BOUNDS.LAT_MIN, latMin - PADDING),
    Math.min(GEO_BOUNDS.LAT_MAX, latMax + PADDING)
  ];
}
```

Reply:
^like that too but not blocking

Reply:
I'll copy-paste jesse's suggestion from the block below.
Thanks for rewriting! I was copy-pasting from the old code.